releases/aws-alb-ingress-controller.yaml

```diff
@@ -92,12 +96,15 @@ releases:
       # choice for the user. This also increases chances charts run on environments with little
       # resources, such as Minikube. If you do want to specify resources, uncomment the following
       # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
-      # limits:
-      #  cpu: 100m
-      #  memory: 128Mi
-      # requests:
-      #  cpu: 100m
-      #  memory: 128Mi
+      #
+      # TODO(tamsky): Should this ^^^ comment disappear ?
```

Comment:
Lets parameterize it like the others. Refer to the other releases for examples

Reply:
Removed.